clients/src/main/java/org/apache/kafka/server/authorizer/Authorizer.java

```diff
@@ -182,114 +171,6 @@ default int aclCount() {
      *                       given type. Return {@link AuthorizationResult#DENIED} otherwise.
      */
     default AuthorizationResult authorizeByResourceType(AuthorizableRequestContext requestContext, AclOperation op, ResourceType resourceType) {
-        SecurityUtils.authorizeByResourceTypeCheckArgs(op, resourceType);
-
-        // Check a hard-coded name to ensure that super users are granted
```

Comment:
I removed comment's as thet seems to be redunant when code is self secriptive by separated methods